templates/base.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  {% block title %}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Taxi Service</title>
+  {% endblock %}
+
+  {% load static %}
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+</head>
+<body>
+  <div class="bg">
+    {% block sidebar %}
+      {% include 'sidebar.html' %}
+    {% endblock %}
+  </div>
+  <div>
+    {% block content %}
+    {% endblock %}
+  </div>
+</body>
+</html>
```

Comment:
Add empty line. Fix in all places where needed

Reply:
done!